iidm/iidm-impl/src/test/java/eu/itesla_project/iidm/network/impl/HvdcLineTest.java

```diff
@@ -1,3 +1,9 @@
+/**
+ * Copyright (c) 2016, All partners of the iTesla project (http://www.itesla-project.eu/consortium)
```

Comment:
Fix copyright data: the copyright info should be "Copyright (c) 2017, RTE (http://www.rte-france.com)" since the end of the european project